manifests/params.pp

```diff
@@ -13,20 +13,31 @@
         $service_file_location   = '/usr/lib/systemd/system/jira.service'
         $service_file_template   = 'jira/jira.service.erb'
         $service_lockfile        = '/var/lock/subsys/jira'
+        $service_systemd         = true
```

Comment:
Its good idea, but I don't know what service providers need to use instead systemd for old RedHat, Debian and for default section?